planty/planty/templates/planty/navbar.html

```diff
@@ -0,0 +1,61 @@
+<!--Logo icon-->
+<nav class="navbar navbar-expand-lg navbar-light">
+    <div class="container">
+        <!--TODO: add logo-->
+        <a class="navbar-brand" href="#">Logo</a>
+
+        <!--dropdown menu-->
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
+            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <!--navbar to the right-->
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+
+                <!--if user authenticated-->
+                <!--TODO authentication-->
+                {% if user.is_authenticated %}
+                <li class="nav-item">
+                    <a class="nav-link" href="#"><i class="bi bi-search-heart nav-icon"></i></a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#"><i class="bi bi-plus-square nav-icon"></i></a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#"><i class="bi bi-chat-square-dots nav-icon"></i></a>
+                </li>
+
+                <!--profile dropdown-->
+                <li>
+                    <div class="dropdown" style="height:100%;">
+                        <button class="dropdown-toggle bi bi-person nav-icon btn-navbar" id="dropdownMenu2"
+                            data-bs-toggle="dropdown" aria-expanded="false"></button>
+
+                        <ul class="dropdown-menu" aria-labelledby="dropdownMenuButton1">
+                            <li><a class="dropdown-item" href="#">My Profile</a></li>
+                            <li><a class="dropdown-item" href="#">Logout</a></li>
+                        </ul>
+                    </div>
+                </li>
+
+                <!--user isn't authenticated-->
+                {% else %}
+                <li class="nav-item">
+                    <nav class="navbar navbar-light">
+                        <div class="container-fluid">
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Login</i></a>
+
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Register</i></a>
+                </li>
+        </div>
+</nav>
+</li>
+{% endif %}
+</ul>
+</div>
+</div>
+</nav>
```

Comment:
I believe these lines at the end make html structure invalid, e.g. `nav` is closed twice.